chart/kubeapps/templates/apprepositories.yaml

```diff
@@ -25,6 +26,12 @@ metadata:
 spec:
   type: helm
   url: {{ .url }}
+  {{- if $root.Values.securityContext.enabled }}
```

Comment:
no need to define a `$root` var here, you can use `$` which points to root directly (e.g. see line 25 above)

Reply:
Neat, didn't know that worked in Go's templating engine. Fixed it